pkg/flare/flare.go

```diff
@@ -127,12 +128,16 @@ func analyzeResponse(r *http.Response, err error) (string, error) {
 	if err != nil {
 		return response, err
 	}
+	if r.StatusCode == http.StatusForbidden {
+		return response, errors.New("HTTP 403 Forbidden: Make sure your API key is valid")
+	}
+
 	b, _ := ioutil.ReadAll(r.Body)
 	var res = flareResponse{}
 	err = json.Unmarshal(b, &res)
 	if err != nil {
 		response = fmt.Sprintf("An unknown error has occurred - Please contact support by email.")
```

Comment:
let's have a more descriptive error here (something like `Error: could not deserialize response body -- Please contact support by email`)